src/toil/jobStores/googleJobStore.py

```diff
@@ -165,20 +165,20 @@ def destroy(self):
         # google freaks out if we call delete multiple times on the bucket obj, so after success
         # just set to None.
         self.bucket = None
+        
+    def _newJobID(self):
+        return "job"+str(uuid.uuid4())
 
-    def create(self, jobNode):
+    def assignID(self, jobDescription):
         jobStoreID = self._newJobID()
-        log.debug("Creating job %s for '%s'",
-                  jobStoreID, '<no command>' if jobNode.command is None else jobNode.command)
-        job = JobGraph.fromJobNode(jobNode, jobStoreID=jobStoreID, tryCount=self._defaultTryCount())
-        if hasattr(self, "_batchedJobGraphs") and self._batchedJobGraphs is not None:
-            self._batchedJobGraphs.append(job)
-        else:
-            self._writeString(jobStoreID, pickle.dumps(job, protocol=pickle.HIGHEST_PROTOCOL))  # UPDATE: bz2.compress(
-        return job
+        log.debug("Assigning ID to job %s for '%s'",
+                  jobStoreID, '<no command>' if jobDescription.command is None else jobDescription.command)
+        jobDescription.jobStoreID = jobStoreID
 
-    def _newJobID(self):
-        return "job"+str(uuid.uuid4())
+    def create(self, jobDescription):
+        # TODO: we don't implement batching, but we probably should.
```

Comment:
Can an issue be created for this?

Reply:
I opened https://github.com/DataBiosphere/toil/issues/3301